src/views/partials/navbar.ejs

```diff
@@ -58,6 +58,16 @@
     </button>
     <div class="collapse navbar-collapse justify-content-end">
         <ul class="navbar-nav">
+          <%if (locals.userIsOrganiser && userIsOrganiser === true) { %>
```

Comment:
Since `locals.userIsOrganiser` is a boolean, I don't think there's any need to explicitly check if it's true. As in, wouldn't  `if (locals.userIsOrganiser)` do the job?

Reply:
Yes, this code can be cleaned up a bit